subsys/bluetooth/host/Kconfig.gatt

```diff
@@ -142,6 +142,14 @@ config BT_GATT_READ_MULTIPLE
 	  This option enables support for the GATT Read Multiple Characteristic
 	  Values procedure.
 
+config BT_GATT_READ_MULTIPLE_VARIABLE_LENGTH
```

Comment:
`CONFIG_BT_EATT` should either `depend on` or `select` this.

Do we have the same issues with Read Long, Write Long and Multiple value notifications?

Reply:
Perhaps we should also have a reference to the table that mandates these features for EATT

Reply:
> CONFIG_BT_EATT should either depend on or select this.

Will add a `select`

> Do we have the same issues with Read Long, Write Long and Multiple value notifications?

Don't think so. Read Multiple, Read Multiple Variable Length, and  Signed Write are the only ones that can be ifdefed out of the `att_handler`s.

> Perhaps we should also have a reference to the table that mandates these features for EATT

Can do.

Reply:
Kind of odd that we have a lot of optional features enabled by default but not the Read Multiple Variable Length one :) 

I guess it's simply due to multiple variable length was added in a later version